docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md

```diff
@@ -44,4 +44,38 @@ options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_in
 dvc_client = DevCycle::DVCClient.new("dvc_server_token_hash", options, true)
 
 user_data = DevCycle::UserData.new({user_id: 'user_id_example'}) # UserData | 
-```
\ No newline at end of file
+```
+
+## Initializing the SDK in a Rails App
+
+The SDK can be initialized in an initializer file:
+
+**Step 1:** Create a new file in `config/initializers` called `devcycle.rb`.
+
+**Step 2:** Add the following code to the `devcycle.rb` file:
+
+```ruby
+options = DevCycle::DVCOptions.new(enable_cloud_bucketing: false, event_flush_interval_ms: 1000, config_polling_interval_ms: 4000)
+Rails.configuration.dvc_client = DevCycle::DVCClient.new(ENV['DVC_SERVER_SDK_KEY'], options, true)
+```
+
+## Initializing the SDK in a Rails App Using Unicorn
+
+When using the `preload_app` configuration with Unicorn, the SDK needs to be initialized in the `after_work` block in the `config/unicorn.rb` file:
```

Comment:
what setting for `preload_app` or would both require the same?

Reply:
it's just when it's set to true. Will clarify